apps/src/studio/studio.js

```diff
@@ -465,6 +491,20 @@ var drawMap = function () {
   titleScreenTextGroup.appendChild(titleScreenTextRect);
   titleScreenTextGroup.appendChild(titleScreenText);
   svg.appendChild(titleScreenTextGroup);
+
+  // Clear rectangle to pick up click events on the visualization
+  var clickRect = document.createElementNS(SVG_NS, 'rect');
+  clickRect.setAttribute('width', Studio.MAZE_WIDTH);
+  clickRect.setAttribute('height', Studio.MAZE_HEIGHT);
+  clickRect.setAttribute('fill-opacity', 0);
+  clickRect.addEventListener('touchstart', function (e) {
```

Comment:
We already have a `Studio.onSvgClicked` - any chance that could be used instead of adding more elements and handlers?